packages/twenty-front/src/modules/object-record/record-board/record-board-card/components/RecordBoardCardBody.tsx

```diff
@@ -46,11 +49,11 @@ export const RecordBoardCardBody = ({
         isRecordReadOnly,
         objectPermissions,
         fieldMetadataId: fieldDefinition.fieldMetadataId,
-        fieldName: fieldDefinition.metadata.fieldName,
-        fieldType: fieldDefinition.type,
-        isCustom: fieldDefinition.metadata.isCustom,
-        objectNameSingular:
-          fieldDefinition.metadata.objectMetadataNameSingular ?? '',
+        fieldMetadataType: fieldDefinition.type,
+        isUIReadOnly:
```

Comment:
let's leave the util get all the parameters and decide rather than composing from the outside, this will likely lead to inconsistencies